compositions/contract/TAC%20-%20Contract%20Template/composition.sqe

```diff
@@ -2037,8 +2040,9 @@ class items
 		class Attributes
 		{
 			name="Ares3_Group";
+			init="[this, ""Ares 3""] call CBA_fnc_setCallsign;";
```

Comment:
https://github.com/CBATeam/CBA_A3/blob/92bf7f0a62955f8b2a8ed35bae60e2360e91bb00/addons/common/fnc_setCallsign.sqf#L38
`setGroupIdGlobal` is global.